From 50b712469c068a960e3b5cb24e5a007852824847 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 17 Jul 2026 16:50:56 +0200
Subject: [PATCH 09291/15333] misc: eeprom: idt_89hpesx: Unify style of
 of_device_id array
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Update the of_device_id array to conform to the most used and generally
recommended coding style. That is:

 - No comma after the list terminator;
 - A comma after an initializer if (and only if) the closing } is not
   directly following;
 - No explicit zeros in the list terminator;
 - A space after an opening { and before a closing }, a single space in
   the list terminator;

Adapt the driver's array accordingly.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/2c35ea101e5eb28334137d9341a8b1e564102ae5.1784299069.git.u.kleine-koenig@baylibre.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/eeprom/idt_89hpesx.c | 104 +++++++++++++++---------------
 1 file changed, 52 insertions(+), 52 deletions(-)

diff --git a/drivers/misc/eeprom/idt_89hpesx.c b/drivers/misc/eeprom/idt_89hpesx.c
index 5bc0294349a0..ea7478cfdd27 100644
--- a/drivers/misc/eeprom/idt_89hpesx.c
+++ b/drivers/misc/eeprom/idt_89hpesx.c
@@ -1445,58 +1445,58 @@ static const struct i2c_device_id idt_ids[] = {
 MODULE_DEVICE_TABLE(i2c, idt_ids);
 
 static const struct of_device_id idt_of_match[] = {
-	{ .compatible = "idt,89hpes8nt2", },
-	{ .compatible = "idt,89hpes12nt3", },
-
-	{ .compatible = "idt,89hpes24nt6ag2", },
-	{ .compatible = "idt,89hpes32nt8ag2", },
-	{ .compatible = "idt,89hpes32nt8bg2", },
-	{ .compatible = "idt,89hpes12nt12g2", },
-	{ .compatible = "idt,89hpes16nt16g2", },
-	{ .compatible = "idt,89hpes24nt24g2", },
-	{ .compatible = "idt,89hpes32nt24ag2", },
-	{ .compatible = "idt,89hpes32nt24bg2", },
-
-	{ .compatible = "idt,89hpes12n3", },
-	{ .compatible = "idt,89hpes12n3a", },
-	{ .compatible = "idt,89hpes24n3", },
-	{ .compatible = "idt,89hpes24n3a", },
-
-	{ .compatible = "idt,89hpes32h8", },
-	{ .compatible = "idt,89hpes32h8g2", },
-	{ .compatible = "idt,89hpes48h12", },
-	{ .compatible = "idt,89hpes48h12g2", },
-	{ .compatible = "idt,89hpes48h12ag2", },
-	{ .compatible = "idt,89hpes16h16", },
-	{ .compatible = "idt,89hpes22h16", },
-	{ .compatible = "idt,89hpes22h16g2", },
-	{ .compatible = "idt,89hpes34h16", },
-	{ .compatible = "idt,89hpes34h16g2", },
-	{ .compatible = "idt,89hpes64h16", },
-	{ .compatible = "idt,89hpes64h16g2", },
-	{ .compatible = "idt,89hpes64h16ag2", },
-
-	{ .compatible = "idt,89hpes12t3g2", },
-	{ .compatible = "idt,89hpes24t3g2", },
-
-	{ .compatible = "idt,89hpes16t4", },
-	{ .compatible = "idt,89hpes4t4g2", },
-	{ .compatible = "idt,89hpes10t4g2", },
-	{ .compatible = "idt,89hpes16t4g2", },
-	{ .compatible = "idt,89hpes16t4ag2", },
-	{ .compatible = "idt,89hpes5t5", },
-	{ .compatible = "idt,89hpes6t5", },
-	{ .compatible = "idt,89hpes8t5", },
-	{ .compatible = "idt,89hpes8t5a", },
-	{ .compatible = "idt,89hpes24t6", },
-	{ .compatible = "idt,89hpes6t6g2", },
-	{ .compatible = "idt,89hpes24t6g2", },
-	{ .compatible = "idt,89hpes16t7", },
-	{ .compatible = "idt,89hpes32t8", },
-	{ .compatible = "idt,89hpes32t8g2", },
-	{ .compatible = "idt,89hpes48t12", },
-	{ .compatible = "idt,89hpes48t12g2", },
-	{ },
+	{ .compatible = "idt,89hpes8nt2" },
+	{ .compatible = "idt,89hpes12nt3" },
+
+	{ .compatible = "idt,89hpes24nt6ag2" },
+	{ .compatible = "idt,89hpes32nt8ag2" },
+	{ .compatible = "idt,89hpes32nt8bg2" },
+	{ .compatible = "idt,89hpes12nt12g2" },
+	{ .compatible = "idt,89hpes16nt16g2" },
+	{ .compatible = "idt,89hpes24nt24g2" },
+	{ .compatible = "idt,89hpes32nt24ag2" },
+	{ .compatible = "idt,89hpes32nt24bg2" },
+
+	{ .compatible = "idt,89hpes12n3" },
+	{ .compatible = "idt,89hpes12n3a" },
+	{ .compatible = "idt,89hpes24n3" },
+	{ .compatible = "idt,89hpes24n3a" },
+
+	{ .compatible = "idt,89hpes32h8" },
+	{ .compatible = "idt,89hpes32h8g2" },
+	{ .compatible = "idt,89hpes48h12" },
+	{ .compatible = "idt,89hpes48h12g2" },
+	{ .compatible = "idt,89hpes48h12ag2" },
+	{ .compatible = "idt,89hpes16h16" },
+	{ .compatible = "idt,89hpes22h16" },
+	{ .compatible = "idt,89hpes22h16g2" },
+	{ .compatible = "idt,89hpes34h16" },
+	{ .compatible = "idt,89hpes34h16g2" },
+	{ .compatible = "idt,89hpes64h16" },
+	{ .compatible = "idt,89hpes64h16g2" },
+	{ .compatible = "idt,89hpes64h16ag2" },
+
+	{ .compatible = "idt,89hpes12t3g2" },
+	{ .compatible = "idt,89hpes24t3g2" },
+
+	{ .compatible = "idt,89hpes16t4" },
+	{ .compatible = "idt,89hpes4t4g2" },
+	{ .compatible = "idt,89hpes10t4g2" },
+	{ .compatible = "idt,89hpes16t4g2" },
+	{ .compatible = "idt,89hpes16t4ag2" },
+	{ .compatible = "idt,89hpes5t5" },
+	{ .compatible = "idt,89hpes6t5" },
+	{ .compatible = "idt,89hpes8t5" },
+	{ .compatible = "idt,89hpes8t5a" },
+	{ .compatible = "idt,89hpes24t6" },
+	{ .compatible = "idt,89hpes6t6g2" },
+	{ .compatible = "idt,89hpes24t6g2" },
+	{ .compatible = "idt,89hpes16t7" },
+	{ .compatible = "idt,89hpes32t8" },
+	{ .compatible = "idt,89hpes32t8g2" },
+	{ .compatible = "idt,89hpes48t12" },
+	{ .compatible = "idt,89hpes48t12g2" },
+	{ }
 };
 MODULE_DEVICE_TABLE(of, idt_of_match);
 
-- 
2.55.0

